src/m365/pa/commands/app/app-list.ts

```diff
@@ -20,8 +26,14 @@ class PaAppListCommand extends AzmgmtItemsListCommand<{ name: string; displayNam
     return ['name', 'displayName'];
   }
 
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.asAdmin = args.options.asAdmin === true;
```

Comment:
We should also track if the `environment` option has been specified as it's optional